spec/features/example_feature_spec.rb

```diff
@@ -0,0 +1,6 @@
+require "spec_helper"
+feature 'capybara and rspec generate passes' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.